classroom.rb

```diff
@@ -0,0 +1,60 @@
+# Represents a classroom with attributes such as label and students
+class Classroom
+  attr_accessor :label, :students
+
```

Comment:
- [ ] The `@students` variable should only have a getter method and not a setter method. It should only be read from outside the class and not set to a new value from outside the class. Kindly implement this change 👍 💻 

`attr_reader :students`

Reply:
@Gambit142 done